website/docs/cli-reference.md

```diff
@@ -98,9 +98,10 @@ Generated links will point to that block explorer. Available values:
 The chain id. It is required unless `--network` is provided.
 #### `--gateway-url STRING`
 The URL of a StarkNet gateway. It is required unless `--network` is provided.
-#### `-i` `--inputs FELT[]`
+#### `-i` `--inputs INPUT[]`
 The inputs to the constructor. Calldata arguments may be of any type that does not contain pointers.
 [Read more about representing Cairo data types in the CLI.](https://www.cairo-lang.org/docs/hello_starknet/more_features.html#array-arguments-in-calldata)
+It is possible to pass the inputs as a list of space-delimited values (`1 2 3`) or as a mapping of their names to their values (`a=12`).
```

Comment:
What is inconsistent about them? Can you specify what you mean, please? `shared argument object`?

Reply:
I think it would be nice if each `--inputs` argument had nearly the same description. Right now each instance uses different words to express same thing, which will highly probably cause diverges in the future.

By `shared argument object` I meant extracting a constant `INPUTS = ProtostarArgument(...)` into `common_argumnets.py` (or probably in this case, introducing a function which will take some arguments in order to slightly customize the argument to each usage place).

Reply:
[address feedback for docs](https://github.com/software-mansion/protostar/pull/1140/commits/259a8d00813769fe9016075045ddc1a16473b53c)